drivers/watchdog/wdt_nrfx.c

```diff
@@ -15,23 +15,34 @@
 
 DEVICE_DECLARE(wdt_nrfx);
 
+/* Macro for calculating watchdog ticks from time in milliseconds.
+ */
+#define WDT_NRF_MS_TO_TICKS(time_ms) ((time_ms * 32768) / 1000)
+
+/* Macro for calculating time in milliseconds from watchdog ticks.
+ */
+#define WDT_NRF_TICKS_TO_MS(ticks) ((ticks * 1000) / 32768)
+
+/* Symbol containing minimum supported timeout time in milliseconds.
+ * Value 0x0F comes from Nordic's Product Specification.
+ */
+#define WDT_NRF_MIN_TIME_MS WDT_NRF_TICKS_TO_MS(0x0FuL)
```

Comment:
Why not just `0xF`?

Reply:
well, I would prefer to work with octets.